theseus_gui/eslintrc.json

```diff
@@ -0,0 +1,33 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true,
+    "node": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:vue/vue3-recommended",
+    "plugin:@typescript-eslint/recommended",
+    "@nuxtjs/eslint-config-typescript",
```

Comment:
can remove this since it's not nuxt 3

Reply:
Thanks for catching, copied ours from knossos and didn't review the contents!

Reply:
Removed.